packages-web/badge/tests/e2e/Objects/input.widget.ts

```diff
@@ -0,0 +1,13 @@
+import page from "../../../../../configs/e2e/src/pages/page";
+
+class InputWidget {
+    get inputElement(): WebdriverIO.Element {
+        const inputWidget = page.getWidget("dataInput");
+        const inputElement = inputWidget.$("input");
+        inputElement.waitForDisplayed();
+        return inputElement;
+    }
+}
+
+const input = new InputWidget();
```

Comment:
This can be done in one line: export default new Page();

Reply:
done